### DISCUSSION TITLE
Buttons in the pause menu trigger their hover effect while other overlays are open

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#19213] [Q&A] [answered]
As title. Not sure if this is intended:

https://user-images.githubusercontent.com/43073074/179473531-03bc6bd3-8db6-4e00-adbe-af57fadf2e23.mp4

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved by https://github.com/ppy/osu/pull/19217

**Comment by ggliv** [comment]
PR has been moved framework-side, follow along here: https://github.com/ppy/osu-framework/pull/5313
